tests/functional/reset_password.js

```diff
@@ -191,19 +191,21 @@ define([
     },
 
     'open complete page with malformed token shows damaged screen': function () {
-      var malformedToken = createRandomHexString(token.length - 1);
       return initiateResetPassword(this, email, 0)
-        .then(openCompleteResetPassword(
-          this, email, malformedToken, code, '#fxa-reset-link-damaged-header'
-        ));
+        .then(function () {
+          var malformedToken = createRandomHexString(token.length - 1);
+          return openCompleteResetPassword(
+            this, email, malformedToken, code, '#fxa-reset-link-damaged-header');
+        });
     },
 
     'open complete page with invalid token shows expired screen': function () {
-      var invalidToken = createRandomHexString(token.length);
       return initiateResetPassword(this, email, 0)
-        .then(openCompleteResetPassword(
-          this, email, invalidToken, code, '#fxa-reset-link-expired-header'
-        ));
+        .then(function () {
+          var invalidToken = createRandomHexString(token.length);
+          return openCompleteResetPassword(
```

Comment:
`openCompleteResetPassword` is meant to be run from a `then`, so I believe here you'll need to stick a () on the end of the call for the function to actually run.